pkg/virt-controller/services/template.go

```diff
@@ -962,8 +966,10 @@ func getNamespaceAndNetworkName(vmi *v1.VirtualMachineInstance, fullNetworkName
 func getNetworkToResourceMap(virtClient kubecli.KubevirtClient, vmi *v1.VirtualMachineInstance) (networkToResourceMap map[string]string, err error) {
 	networkToResourceMap = make(map[string]string)
 	for _, network := range vmi.Spec.Networks {
-		if network.Multus != nil {
-			namespace, networkName := getNamespaceAndNetworkName(vmi, network.Multus.NetworkName)
+		var namespace string
```

Comment:
why should these variables be defined in this scope and not under ```if``` from line 971 below?

Reply:
done